app/backend/src/clj/teet/integration/integration_email.clj

```diff
@@ -58,13 +57,13 @@
              "\r\n\r\n"
              (->b64 body))))))
 
-(defn- send-email!*
-  "Invoke email sending"
+(defn- with-smtp-config [smtp-conf msg]
+  (assoc msg :server smtp-conf))
+
+(defn send-email-smtp!*
   [msg]
-  (aws/invoke
-   @email
-   {:op :SendRawEmail
-    :request {:RawMessage {:Data (.getBytes (raw-message msg) "UTF-8")}}}))
+  (let [smtp-node-config (:server msg)]
```

Comment:
You should pass server config as separate parameter, not assoc it to the message